src/SignalR/clients/ts/signalr/src/WebSocketTransport.ts

```diff
@@ -51,7 +54,7 @@ export class WebSocketTransport implements ITransport {
             let opened = false;
 
             if (Platform.isNode) {
-                const headers = {};
+                const { headers } = this;
```

Comment:
Since `headers` is being mutated, it should probably be shallow copied. E.g.

```suggestion
                const headers = { ...this.headers };
```

Nit: I also think defaultHeaders is a more descriptive name.